server/modules/user/api/userApi.js

```diff
@@ -9,6 +9,30 @@ const SystemError = require('../../../../server/utils/systemError')
 
 module.exports.init = app => {
 
+  // ==== CREATE
+
+  app.post('/survey/:surveyId/users/invite', AuthMiddleware.requireUserInvitePermission, async (req, res, next) => {
+    try {
+      const surveyId = Request.getRestParam(req, 'surveyId')
+      const email = Request.getRestParam(req, 'email')
+      const groupId = Request.getRestParam(req, 'groupId')
+
```

Comment:
Maybe it's easier to use Request.getParams
E.g. const { surveyId, email, groupId } = Request.getParams(req)

Reply:
Done